frontend/public/components/deployment.tsx

```diff
@@ -91,7 +92,18 @@ const DeploymentDetails: React.FC<DeploymentDetailsProps> = ({obj: deployment})
     <div className="co-m-pane__body">
       <SectionHeading text="Deployment Overview" />
       {deployment.spec.paused && <WorkloadPausedAlert obj={deployment} model={DeploymentModel} />}
-      <DeploymentPodCounts resource={deployment} resourceKind={DeploymentModel} />
+      <TopologyDataController
+        namespace={deployment.metadata.namespace}
+        application={deployment.metadata.labels['app.kubernetes.io/part-of']}
+        knative={false}
```

Comment:
We need knative info somehow plugged in through extensions so that we can display a scaled to zero visual.